cohortextractor/ons_cis_utils.py

```diff
@@ -79,7 +79,7 @@
     "travel_abroad": "bool",
     "travel_abroad_date": "date",
     "visit_date": "date",
-    "visit_id": "str",
+    "pseudo_visit_id": "bytes",
```

Comment:
Does this definitely work?  I remember not being able to add certain types here because it causes pandas errors in the converting for output

Reply:
We don't extract `pseudo_visit_id`, so I don't think Pandas will error; I hope the tests would tell me, if not. Did you have a test in mind?

Reply:
Actually, I expect it does in the current tests, because it's only being used for tie-breaking.  I think it will break if you try to return `pseudo_visit_id` in the output (e.g. in `tests/test_tpp_backend.py:test_ons_cis`)